sdkproject/Assets/Mapbox/Examples/Scripts/TileErrorHandler.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using Mapbox.Unity.Map;
+using UnityEngine;
+
+/// <summary>
+/// Monobehavior Script to handle TileErrors. 
+/// There's an OnTileError event on AbstractMapVisualizer, AbstractTileFactory and UnityTile classes that one can subscribe to to listen to tile errors
+/// </summary>
+public class TileErrorHandler : MonoBehaviour {
+
+
+	void OnEnable () {
+		GetComponent<AbstractMap>().MapVisualizer.OnTileError += _OnTileErrorHandler;
```

Comment:
Prefer `[SerializeField`] for additional developer flexibility--only get component if assigned reference is null.

Reply:
Done!